src/config.coffee

```diff
@@ -1,3 +1,9 @@
+Languages = require('./languages/')
+
+langFallbackDefault = '- none (do not format) -'
+langFallbacks = new Languages().languageNames
```

Comment:
Fortunately, by creating a new `Languages` instance, we have access to all of the loaded `languages` and their `name` property.

Instead, let's try something like:

```coffee
# Add lodash to the requirements at the top, since it is not already
_ = require('lodash')
# Map from the Languages to each language.name
langFallbacks = _.map((new Languages()).languages, (language) => language.name)
```

It essentially is the opposite of how the `getLanguages` works, see https://github.com/Glavin001/atom-beautify/blob/master/src/languages/index.coffee#L85 , since it gets the language `name`s the same way.